### DISCUSSION TITLE
文档中的跳转到 ProComponents 的"更多"导航怎么不见了

### DISCUSSION THREAD

**Discussion by ByYogi** [ant-design/ant-design#46976] [General]
![image](https://github.com/ant-design/ant-design/assets/111735598/f3dde59f-5cb1-4bf6-a666-ecd6aaa63d35)
![image](https://github.com/ant-design/ant-design/assets/111735598/34251488-99b3-476a-8f36-7f2e6ef52822)

**Comment by afc163** [comment]
可以从底部 footer 进入

**Comment by ByYogi** [comment]
好的谢谢
![image](https://github.com/ant-design/ant-design/assets/111735598/6b81ae37-b721-4d8a-97bf-9147735e5a75)
